src/sparseml/pytorch/image_classification/utils/helpers.py

```diff
@@ -528,13 +572,7 @@ def _download_model_from_zoo_using_recipe(
             f" but got {recipe_stub} instead"
         )
 
-    files = Zoo.download_recipe_base_framework_files(
-        stub=recipe_stub,
-        extensions=[".pth"],
-    )
-
-    checkpoint_path = files[0]
-    return checkpoint_path
+    return Model(recipe_stub).training.default.get_file("model.pth").path
```

Comment:
noting again here - we need to refactor to make sure .ckpt files are accounted for for transfer recipes